en/developer_guide/elasticsearch_indexation.rst

```diff
@@ -0,0 +1,116 @@
+Elasticsearch indexation
+========================
+
+Elasticsearch
+-------------
+
+`Elasticsearch`_ is a common indexor based on the `lucene`_ engine written in Java. To install it on the
+computer running the application, you should add the ``indexation`` group to the file
+``provisioning/hosts/vagrant`` (or the environment you are working on):
```

Comment:
Vagrant is not a prerequisite to dev with Open Orchestra. Maybe there should be two sections: one with vagrant, one without, such as in the installation guide